services/blockchain-indexer/tests/unit/parser.test.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const {
+	parseToJSONCompatObj,
+	parseInputBySchema,
+} = require('../../shared/utils/parser');
+
+const {
+	liskAccount,
+	liskBlock,
+	liskBlockHeader,
+	liskBlockHeaderAsset,
+	liskBlockHeaderAssetV3,
+	liskTransaction,
+	liskTransactionAssets,
+} = require('../constants/parser');
+
+describe('Unit tests for parse utilities', () => {
+	describe('parseToJSONCompatObj', () => {
+		it('Parse buffer', async () => {
```

Comment:
Please adjust the others as well.
```suggestion
		it('should parse buffer', async () => {
```